Mathlib/Geometry/Manifold/Instances/Sphere.lean

```diff
@@ -575,3 +577,87 @@ theorem contMDiff_circleExp : ContMDiff 𝓘(ℝ, ℝ) (𝓡 1) ∞ Circle.exp :
 @[deprecated (since := "2024-07-25")] alias contMDiff_expMapCircle := contMDiff_circleExp
 
 end Circle
+
+section OnePoint
+
+/-
+# One-point compactification of Euclidean space is homeomorphic to the sphere.
+-/
+
+variable {n : ℕ}
+variable {v : EuclideanSpace ℝ (Fin n.succ)}
+variable (hv : ‖v‖ = 1)
+variable (hv' : v ∈ sphere 0 (1:ℝ))
+/-
+    For example, `v` could be
+    `(EuclideanSpace.single (0:Fin n.succ) (1:ℝ))`
+-/
+
+open Set Submodule
+
+/-- The inverse stereographic projection is injective. -/
+theorem stereo_inj : Injective (stereoInvFun hv) := fun x y h =>
```

Comment:
Please move this close to the definition of the stereographic projection.

Reply:
Done.